tests/integration/integration_python_test.go

```diff
@@ -1769,3 +1769,29 @@ func TestDynamicProviderPython(t *testing.T) {
 		})
 	}
 }
+
+// The shutdown of the callback server used for transform would log exceptions
+// to stderr.
+// https://github.com/pulumi/pulumi/issues/18176
+//
+//nolint:paralleltest // ProgramTest calls t.Parallel()
+func TestRegress18176(t *testing.T) {
+	stdout := &bytes.Buffer{}
+	stderr := &bytes.Buffer{}
+	integration.ProgramTest(t, &integration.ProgramTestOptions{
+		Env: []string{"PULUMI_DISABLE_AUTOMATIC_PLUGIN_ACQUISITION=false"},
```

Comment:
I thought this was supposed to be set by default :thinking: (not that it hurts much setting it again here, just curious why this was needed)

Reply:
This is the reverse, it disables the disable

Reply:
Ah doh of course! Makes sense!